### ISSUE TITLE
[BugReport]cannot play videos

### ISSUE THREAD

**Issue by android-notes** [media-kit/media-kit#528]
### Version:  

  media_kit: ^1.1.8
  media_kit_video: ^1.2.1                        # For video rendering.
  media_kit_libs_video: ^1.0.3                   # Native video dependencies.

### Platform
macOS 13.2 M1 

### Code
`player.open(Media(assetVideo)); // load from assets`

### Bug
release package cannot play videos, it keeps loading, while the debug package works fine. 

Even after compiling using the 'arch -x86_64 flutter build macos' command, the issue persists on the M1. The error logs during video loading are as follows.

`flutter: media_kit: wakelock: _count = 0
MediaKitEventLoopHandler::Register: std::thread exit: 5185330304
MediaKitEventLoopHandler::Dispose: 5185330304
2023-10-11 17:40:15.314058+0800  MyApp[83001:35615236] VideoOutput: enableHardwareAcceleration: true
flutter: VideoOutput.Resize
flutter: {handle: 5185315024, id: 105553147397184, rect: {width: 0.0, top: 0.0, left: 0.0, height: 0.0}}
flutter: NativeVideoController: Texture ID: 105553147397184
flutter: media_kit: wakelock: _count = 1
`

**Comment by alexmercerind** [comment]
Did you add the assets to your pubspec.yaml?

**Comment by android-notes** [comment]
Yes, I have already placed them in the assets. However, I still cannot play the videos in the release package. The videos play correctly only in the debug package. In the release directory on Windows, I can indeed see these video files located at build\windows\runner\Release\data\flutter_assets\assets\videos.  @alexmercerind

**Comment by android-notes** [comment]
This is a simple demo that can be reproduced on Windows 10.    [github](https://github.com/android-notes/media_kit_bug)

**Comment by alexmercerind** [comment]
I can play the asset on Microsoft Windows; in both Release & Debug mode.

**Comment by alexmercerind** [comment]
@android-notes 

On macOS, I had to add the required permissions: https://github.com/media-kit/media-kit#macos

After this, I could play the asset on macOS; both in Release & Debug mode.
